src/firefox/package.json

```diff
@@ -7,7 +7,6 @@
   "license": "MPL 2.0",
   "version": "0.1",
   "permissions": {
-    "private-browsing": true,
-    "unsafe-content-script": true
+    "private-browsing": true
```

Comment:
I think we don't need this either.

We used to do login flow in private window, but we don't anymore, or if we do we shouldn't because we're using account chooser. Can you remove it while you're at it?